baseMdl/mdl.py

```diff
@@ -67,14 +12,25 @@ class Model:
     """
 
     def __init__(self, mdl_name, environment):
+        # Name and Env
         self.mdlName = mdl_name
         self.env = environment
+        # The model itself
+        self.q_table = None
+        # For discretizing
         self.upper_bounds = None
         self.lower_bounds = None
         self.buckets = None
-        self.q_table = None
-
-    def discretize(obs, lower_bounds, upper_bounds, buckets):
+        # For training
+        self.episodes = None
+        self.alpha = None
+        self.gamma = None
+        # For training exploration
+        self.min_epsilon = None
+        self.max_epsilon = None
+        self.decay = None
```

Comment:
Para que funcione lo de salvar a un directorio en específico, aquí se indica el nombre de dicho directorio:
```suggestion
        self.decay = None
        # For saving and loading models
        self.models_dir_name = "savedModels"
```